Backend/Controllers/HealthController.cs

```diff
@@ -0,0 +1,21 @@
+using Microsoft.AspNetCore.Mvc;
+
+namespace biletmajster_backend.Controllers;
+
+[ApiController]
+[Route("[controller]")]
+public class HealthController : ControllerBase
+{
+    private readonly ILogger<HealthController> _logger;
+
+    public HealthController(ILogger<HealthController> logger)
+    {
+        _logger = logger;
+    }
+
+    [HttpGet(Name = "GetWeatherForecast")]
+    public string Get()
+    {
+        return $"{DateTime.Now} OK";

```

Comment:
Preferably JSON instead of literal string

Reply:
Fixed